Platform/CoffeelakeBoardPkg/CfgData/CfgData_Power.dsc

```diff
@@ -0,0 +1,73 @@
+## @file
+#
+#  Copyright (c) 2018 - 2019, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+  # !BSF PAGES:{PWR:SIL:"POWER"}
+  # !BSF PAGE:{PWR}
+  # !BSF SUBT:{CFGHDR_TMPL:POWER_CFG_DATA:0:0}
+  # !HDR EMBED:{POWER_CFG_DATA:TAG_610:START}
+
+  # !BSF NAME:{1-Core Ratio Limit}
+  # !BSF TYPE:{EditNum, HEX, (0,0xFF)}
+  # !BSF HELP:{1-Core Ratio Limit: LFM to Fused, For overclocking part: LFM to 255. This 1-Core Ratio Limit Must be greater than or equal to 2-Core Ratio Limit, 3-Core Ratio Limit, 4-Core Ratio Limit, 5-Core Ratio Limit, 6-Core Ratio Limit, 7-Core Ratio Limit, 8-Core Ratio Limit. Range is 0 to 255}
+  gCfgData.OneCoreRatioLimit                 |      * | 0x01 | 0x2C
```

Comment:
These default values are the same for CFL-R? No delta required?

Reply:
updated upd's for all 3 platforms in new patch